addons/io_scene_gltf2/io/com/gltf2_io_constants.py

```diff
@@ -161,3 +161,6 @@ class BufferViewTarget(IntEnum):
 GLTF_DATA_TYPE_MAT4 = "MAT4"
 
 GLTF_IOR = 1.5
+
+# Rounding digit used for normal rounding
+NORMALS_ROUNDING_DIGIT = 6
```

Comment:
No new line at end of file.

What text editor do you use? You can configure most editors to automatically strips trailing spaces from all lines and ensure there's a new line at EOF on save.

Reply:
Seems VSCode on one of the computers I use was not set correctly.
Fixed